src/OrleansProviders/OrleansProviders.csproj

```diff
@@ -14,5 +14,6 @@
   <ItemGroup>
     <ProjectReference Include="..\Orleans.Core.Abstractions\Orleans.Core.Abstractions.csproj" />
     <ProjectReference Include="..\Orleans.Core\Orleans.Core.csproj" />
+    <ProjectReference Include="..\Orleans.Runtime\Orleans.Runtime.csproj" />
```

Comment:
I thought OrleansProvider shouldn't have an dependency on Orleans runtime.

Reply:
It would make Orleans.Client has a transient dependency on Orleans.Runtime.

Reply:
Good catch. Looks like we need a separate change to avoid this dependency that would have to go in before this one.